apps/server/lib/http/facades/auth.ts

```diff
@@ -45,23 +47,23 @@ export class AuthFacade extends QueryFacade {
 
 		// Try and load the user with attached org data, otherwise load them without it.
 		// TODO: Fix our broken queries so that we can optionally get linked data
-		let user = await this.queryAPI(
-			context,
-			session,
-			schema,
-			{
-				limit: 1,
-			},
-			ipAddress,
-		).then((elements) => {
-			return elements[0] || null;
-		});
+		let user = (
+			await this.queryAPI(
+				context,
+				session,
+				schema,
+				{
+					limit: 1,
+				},
+				ipAddress,
+			)
+		)[0] as Contract | null;
```

Comment:
`queryAPI` is an internal method. Why isn't it returning the right type?

Reply:
```typescript
const a: Contract[] = [];
const b = a[0];
```

Type of `b` is `Contract`, but I expected `Contract | undefined`. This is the reason why removing cast causes an error.

Reply:
https://stackoverflow.com/questions/50647399/typescript-accessing-an-array-element-does-not-account-for-the-possibility-of-u

Reply:
:thinking:

Reply:
@Ereski adding  `--noUncheckedIndexedAccess` sounds good for me, what do you think?

Reply:
Sounds good to me too! And in `jellyfish-config` if it's possible too!

Reply:
@Ereski https://github.com/product-os/jellyfish-config/pull/60